units/provisioning_create_host.go

```diff
@@ -147,8 +147,13 @@ func (j *createHostJob) Run(ctx context.Context) {
 
 		allRunningDynamicHosts, err := host.CountAllRunningDynamicHosts()
 		j.AddError(err)
+		smallDistroException := false
```

Comment:
I think we can just have this always be on. I can't think of a scenario where we'd want to keep distros with 0 running hosts as-is if there are tasks in the queue

Reply:
The host number is defined in L127 as `numHosts, err = host.CountRunningHosts(j.host.Distro.Id)`. So this should apply for all distros where the host number is low enough. I renamed this variable which should hopefully make this clearer.